setup.py

```diff
@@ -23,7 +23,8 @@
     install_requires=[
         'futures',
         'protobuf>=3.0.0b2.post2',
-        'opentracing>=1.2.1,<1.3'
+        'opentracing>=1.2.1,<1.3',
+        'six<2.0',
```

Comment:
Just to double check, is it also true that you need at least `1.10.0`? Should it be `>=1.10.0,<2.0` or does any `1.X` version work?